### PR TITLE
Switched from unmaintained and vulnerable package

### LINKED ISSUE #192: Prototype Pollution vulenrability
### Which SDK version are you using?
^12.0.1

### What's the issue?
Snyk reporting Prototype Pollution vulenrability

### Steps/Sample code to reproduce the issue
https://app.snyk.io/test/npm/facebook-nodejs-business-sdk/12.0.1

#### Observed Results:
snyk_issue

#### Expected Results:
No vulnerabilities should be found by Snyk

### PR DESCRIPTION
**Pull request by finnmich**
Switched iso-3166-1-alpha-2 to iso-3166-1. Closes #192

### CODE REVIEW

File: src/objects/serverside/utils.js [hunk 1]
```diff
@@ -97,7 +97,7 @@ export default class ServerSideUtils {
    * @return {String} Normalized ISO country code.
    */
   static normalizeCountry (country: string) {
-    if (country_codes.getCountry(country.toUpperCase()) == null) {
+    if (!country_codes.whereAlpha2(country)) {) {
```
[the remark this answers is not in the document]
Comment (finnmich) [reply src/objects/serverside/utils.js]: Yes... that's what I get when I edit without an IDE, sorry :)
Fixed now.

**Comment by jfaylon** [comment]
Hopefully this code fixes this: https://github.com/advisories/GHSA-pc58-wgmc-hfjr

**Comment by cassioscofield** [comment]
Hey guys is there some reason why this has not been merged yet?
I'd hope to get this vulnerability fixed sooner rather than later.

**Comment by hagar-dolev** [comment]
@JollyTeo Is there an update on merging this PR?

**Comment by JollyTeo** [comment]
@hagar-dolev I don't have permissions to merge. I ping some of the last contributors to this project to see if some good hearted can help moving this forward, Sorry to bother @supasate @jingping2015 @davit-y

**Comment by JollyTeo** [comment]
@justiwong as I said above, I don't have permission to merge. Only people from Facebook can @facebook-github-bot 
Y'all need to ping them not me :)

**Comment by andrew-ignatiev** [comment]
Hey @finnmich, do you know when new version which includes this fix will be published to npm ?

**Pull request closed without merging**